src/azul/__init__.py

```diff
@@ -430,13 +424,18 @@ def drs_endpoint(self):
     def lambda_names(self) -> List[str]:
         return ['indexer', 'service']
 
-    @property
-    def indexer_name(self) -> str:
-        return self.qualified_resource_name('indexer')
+    def indexer_name(self, lambda_name: Optional[str] = None) -> str:
+        return self._app_name('indexer', lambda_name)
 
-    @property
-    def service_name(self) -> str:
-        return self.qualified_resource_name('service')
+    def service_name(self, lambda_name: Optional[str] = None) -> str:
+        return self._app_name('service', lambda_name)
+
+    def _app_name(self, app: str, lambda_name: Optional[str]):
```

Comment:
this method

```suggestion
    def _function_name(self, lambda_name: str, handler_name: str):
```

If you have a better idea, let me know.

Reply:
I like this.